man/needrestart.1

```diff
@@ -52,6 +52,9 @@ nagios plugin mode: makes output and exit codes nagios compatible
 \fB\-f\fR <fe>
 override debconf(7) frontend, sets the DEBIAN_FRONTEND environment variable to <fe>
 .TP
+\fB\-t\fR <number>
+When checking running scripts, allow process start times that are close to script file timestamps, within this tolerance (default 0). Using a value of 2 is suggested for automated checks of Linux hosts, on which system limitations prevent more accurate measurements of process start times. Using this option may prevent false positives yet cause false negatives.
```

Comment:
Please add a hint about the unit (seconds).

Reply:
Fixed.